### PR TITLE
Fix Advanced Filter search bug

### PR DESCRIPTION
**Pull request by wyukawa**
If I fill in project field and push Filter button in Advanced Filter, search doesn't work.

Here is azkaban web server log
```
java.sql.SQLException: Column 'enc_type' in field list is ambiguous Query: SELECT exec_id, enc_type, flow_data FROM execution_flows  ef JOIN projects p ON ef.project_id = p.id WHERE name LIKE ?  ORDER BY exec_id DESC LIMIT ?, ? Parameters: [%hoge%, 0, 16]
        at org.apache.commons.dbutils.AbstractQueryRunner.rethrow(AbstractQueryRunner.java:363)
        at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:350)
        at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:288)
        at azkaban.db.DatabaseOperator.query(DatabaseOperator.java:65)
        at azkaban.executor.ExecutionFlowDao.fetchFlowHistory(ExecutionFlowDao.java:213)
        at azkaban.executor.JdbcExecutorLoader.fetchFlowHistory(JdbcExecutorLoader.java:152)
        at azkaban.executor.ExecutorManager.getExecutableFlows(ExecutorManager.java:659)
        at azkaban.webapp.servlet.HistoryServlet.handleHistoryPage(HistoryServlet.java:114)
        at azkaban.webapp.servlet.HistoryServlet.handleGet(HistoryServlet.java:60)
        at azkaban.webapp.servlet.LoginAbstractAzkabanServlet.doGet(LoginAbstractAzkabanServlet.java:123)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:668)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:770)
        at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
        at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:401)
        at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
        at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:766)
        at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
        at org.mortbay.jetty.Server.handle(Server.java:326)
        at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
        at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:928)
        at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:549)
        at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
        at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
        at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
        at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:582)
```

### CODE REVIEW

**Comment by wyukawa** [comment]
Looks like it overlaps with #873 #915

**Comment by wyukawa** [comment]
@kunkun-tang @HappyRay Can you take a look at this pull request when you get a chance?
I guess this is a simple issue.

**Comment by HappyRay** [comment]
@wyukawa 
Thanks.
Someone will get back to you. 
Please see my comments for the referenced PRs. e.g. unit tests.

**Comment by reallocf** [comment]
Hey @wyukawa,

This PR looks good to me - if you could provide a unit test as Ray suggested then I'd be happy to give a ship-it and merge it.

Thanks!!

**Comment by wyukawa** [comment]
@HappyRay @reallocf Thank you for comments!
I add unit test.
please check.

**Comment by wyukawa** [comment]
@HappyRay @reallocf Could you please check?

**Comment by HappyRay** [comment]
@wyukawa sorry about the delay.

@ameyamk I am too busy to do PR reviews. If secondary oncalls can't handle the load too, could you assign someone? 
Thanks

File: azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java [hunk 1]
```diff
@@ -268,7 +268,7 @@ public ExecutableFlow fetchExecutableFlow(final int execId) throws ExecutorManag
       ResultSetHandler<List<ExecutableFlow>> {
 
     static String FETCH_BASE_EXECUTABLE_FLOW_QUERY =
-        "SELECT exec_id, enc_type, flow_data FROM execution_flows ";
+        "SELECT exec_id, ef.enc_type, flow_data FROM execution_flows ";
```
[the remark this answers is not in the document]
Comment (wyukawa) [reply azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java]: @kunkun-tang Thank you for comment!
I fixed in https://github.com/azkaban/azkaban/pull/1638/commits/e300bc90f67ff1f4247f154b5006905e51fcdfe1

**Comment by wyukawa** [comment]
@kunkun-tang hmm, test passes in my local environment.
What can I do?

**Comment by wyukawa** [comment]
hmm, CI doesn't start...

**Comment by wyukawa** [comment]
hmm, CI doesn't work well...

**Comment by wyukawa** [comment]
I don't know why CI failed...
```
azkaban.metric.MetricManagerTest > managerEmitterHandlingTest FAILED
    java.lang.AssertionError: Failed to report metric expected:<1> but was:<0>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at azkaban.metric.MetricManagerTest.managerEmitterHandlingTest(MetricManagerTest.java:92)
```

My forked repository pass tests.
https://travis-ci.org/wyukawa/azkaban/builds/364027602

Any ideas?

**Comment by HappyRay** [comment]
@wyukawa  and @kunkun-tang 
Thanks

**Comment by wyukawa** [comment]
Thanks!

**Pull request merged**